SimonImanigirimpuhwe/my-brand-website-api
# #4 chore(mantain): add eslint for linting

#### What does this PR do?

- Setting up Eslint

#### Description of Task to be completed?

- Install eslint 

- Add prettier config file

- Add eslint config file


#### How should this be manually tested?

- Clone the repo 

- In your home directory  cd this branch `ch-setting-up-Eslint-#4`

- Then run `npm install` to install all dependencies

> You should see highlighted lines that are breaking the eslint defined rules

#### Any background context you want to provide?

- To be able to see the linting lines you need to install eslint extension in your code editor

#### What are the relevant Trello stories?

- [#4](https://trello.com/c/Px4oFLx3/4-setting-up-eslint)

#### Screenshots (if appropriate)

- N/A

#### Questions:

- N/A
